docs/cheatsheet/functions.rst

```diff
@@ -0,0 +1,25 @@
+.. _ref_cheatsheet_functions:
+
+Functions
+=========
+
+Define a function for counting reviews given a user name:
+
+.. code-block:: edgeql
+
+    CREATE FUNCTION review_count(name: str) -> int64
+    USING EdgeQL $$
+        WITH MODULE default
+        SELECT count(
+            (
+                SELECT Review
+                FILTER .author.name = name
+            )
+        )
+    $$
+
+Drop a user-defined function:
+
+.. code-block:: edgeql
+
+    DROP FUNCTION review_count(name: str);
```

Comment:
Mention polymorphic functions -- define a function that can accept both ints and floats as arguments.

Reply:
OK